src/multidim/intervalbox.jl

```diff
@@ -73,12 +73,12 @@ diam(X::IntervalBox) = maximum(diam.(X.v))
 
 emptyinterval(X::IntervalBox{N,T}) where {N,T} = IntervalBox(emptyinterval.(X.v))
 
-isinf(X::IntervalBox) = any(isinf(X))
+isinf(X::IntervalBox) = any(isinf.(X))
```

Comment:
This can be

any(isinf, X)

(with a comma) which should be (slightly) faster.